package.json

```diff
@@ -31,7 +31,7 @@
     "gridstack": "^1.1.0"
   },
   "devDependencies": {
-    "@ember/jquery": "^0.5.2",
+    "@ember/jquery": "^1.1.0",
```

Comment:
Can we remove?

Reply:
Also turn off jquery here
https://github.com/yahoo/ember-gridstack/blob/a5187d85da9067c094b60a19a4977ec430e42983/tests/dummy/config/optional-features.json#L4

Do we still need this too?
https://github.com/yahoo/ember-gridstack/blob/e429d659187a0bd01919b7e5332a997cc53bc860/package.json#L60

Reply:
Updated!